assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -108,6 +108,12 @@ export default {
                 ? "urgent"
                 : "unread";
             }
+
+            get contentCSSClass() {
+              return this.channel.current_user_membership.muted
+                ? "--muted"
```

Comment:
This is kind of odd, should there be a prefix for the class?

Reply:
![Screenshot from 2022-10-12 14-59-32](https://user-images.githubusercontent.com/4335742/195272230-7b8409d4-22c3-4f96-80cb-76287027c4cf.png)

This definitely feels off because we usually don't name our CSS classes like this.

Reply:
Good point, will fix

Reply:
No worries I fixed it in https://github.com/discourse/discourse-chat/commit/aa5f731f1d426fca902a2c5370646f4052cbf0c7v yesterday while looking at something related.